packages/lime-proto-babeld/files/usr/lib/lua/lime/proto/babeld.lua

```diff
@@ -103,17 +103,54 @@ function babeld.setup_interface(ifname, args)
 	local vlanProto = args[3] or "8021ad"
 	local nameSuffix = args[4] or "_babeld"
 
+	local addIPtoIf = true
+
+	--! If Babeld is without VLAN (vlanId is 0) it cannot run directly
+	--! on ethernet interfaces which are inside of a bridge (e.g. eth0 or eth0.1)
+	--! because they cannot have an IPv6 Link-Local, so Babeld has to run on 
+	--! the bridge interface br-lan
+	--! If Babeld's Hello packets run over Batman-adv (whose bat0 is also 
+	--! included in br-lan), the links will have a wrong quality metric,
+	--! so these hello on bat0 have to be filtered
```

Comment:
It is not that they have a wrong quality metric, it's that babeld will see all nodes as direct neighbours through batman

Reply:
mh... Isn't this the same? But ok, I'll change the text :)

Reply:
no it is not the same